contracts/acl/ACL.sol

```diff
@@ -421,14 +422,18 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
 
         // a raw call is required so we can return false if the call reverts, rather than reverting
         bytes memory checkCalldata = abi.encodeWithSelector(sig, _who, _where, _what, _how);
-        uint256 oracleCheckGas = ORACLE_CHECK_GAS;
 
         bool ok;
-        assembly {
-            ok := staticcall(oracleCheckGas, _oracleAddr, add(checkCalldata, 0x20), mload(checkCalldata), 0, 0)
+
+        if (gasleft() > ERROR_GAS_ALLOWANCE) {
+            uint256 oracleCheckGas = gasleft() - ERROR_GAS_ALLOWANCE;
+            assembly {
+                ok := staticcall(oracleCheckGas, _oracleAddr, add(checkCalldata, 0x20), mload(checkCalldata), 0, 0)
+            }
         }
 
         if (!ok) {
+            require(gasleft() > ERROR_GAS_ALLOWANCE, ERROR_ORACLE_OOG);
```

Comment:
As mentioned in the call, we should use `max(gasleft / 64, ERROR_GAS_ALLOWANCE)` as the comparison value here.